code/game/objects/items/weapons/mop.dm

```diff
@@ -23,15 +23,15 @@
 
 
 obj/item/weapon/mop/proc/clean(turf/A)
-	if(reagents.has_reagent("water", 1) || reagents.has_reagent("holywater", 1) || reagents.has_reagent("vodka", 1))
+	if(reagents.has_reagent("water", 1) || reagents.has_reagent("holywater", 1) || reagents.has_reagent("vodka", 1) || reagents.has_reagent("cleaner", 1))
 		A.clean_blood()
 		for(var/obj/effect/O in A)
 			if(is_cleanable(O))
 				qdel(O)
 		if(istype(A, /turf/closed))
 			var/turf/closed/C = A
 			C.thermite = 0
-	reagents.reaction(A, TOUCH, 10)	//10 is the multiplier for the reaction effect. probably needed to wet the floor properly.
+	reagents.reaction(A, TOUCH, 1)	//1 used 1 applied.
```

Comment:
The mop actually currently works as such that when it has one unit of water, holywater or vodka it cleans the floor. I added cleaner to the list because it used to only work because the mop would apply one unit of cleaner.